cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -969,25 +969,33 @@ data:
             proxy_set_header  X-Real-IP  $remote_addr;
             proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
         }
-        location = /model/cloudCost/integration/export {
+        location = /model/cloud/config/export {
```

Comment:
so was the FE broken? or will they need to impl these new paths?

Reply:
do they use any of these endpoints you modified?

Reply:
just confirmed that the modified endpoints are not currently used